src/plugins/intel_cpu/tests/functional/shared_tests_instances/snippets/add.cpp

```diff
@@ -21,14 +21,36 @@ INSTANTIATE_TEST_SUITE_P(smoke_Snippets_Eltwise, Add,
                              ::testing::Values(CommonTestUtils::DEVICE_CPU)),
                      Add::getTestCaseName);
 
+
+namespace snippets_static_1 {
+// These  inputs are needed to test static Loop optimizations (emit the whole tile, body with increments, set WA etc)
+std::vector<ov::Shape> inShapesStatic1{{1, 16, 29,  1}, {1, 16, 29,  7}, {1, 16, 29,  8}, {1, 16, 29,  15}, {1, 16, 29,  16}, {1, 16, 29,  31}};
+std::vector<ov::Shape> inShapesStatic2{{1, 16, 29,  1}, {1, 16, 1, 1}, {1, 1, 1, 1}};
 INSTANTIATE_TEST_SUITE_P(smoke_Snippets_Eltwise, AddSinh,
-        ::testing::Combine(
-        ::testing::Values(ov::Shape {1, 42, 16, 64}),
-        ::testing::Values(ov::Shape {1, 42, 16,  1}),
-        ::testing::Values(3), // Add + 2 converts after inputs
-        ::testing::Values(1), // Subgraph is created, since the inputs are followed by converts
-        ::testing::Values(CommonTestUtils::DEVICE_CPU)),
+                         ::testing::Combine(
+                             ::testing::ValuesIn(inShapesStatic1),
+                             ::testing::ValuesIn(inShapesStatic2),
+                             ::testing::Values(3), // Add + 2 converts after inputs
+                             ::testing::Values(1), // Subgraph is created, since the inputs are followed by converts
+                             ::testing::Values(CommonTestUtils::DEVICE_CPU)),
                          AddSinh::getTestCaseName);
+// test cross-tile (vector vs scalar) optimizations in the absence of vector tile
+std::vector<std::vector<ov::Shape>> inShapesStatic{
+        {{1, 128, 1, 1}, {1, 128, 1, 1}},
```

Comment:
Could you please add some test like `[1,128,30,30] and [1,128,1,30]` to check for finalization offsets?

Reply:
Sure, added two test cases:
        {{1, 128, 9, 30}, {1, 128, 1, 30}},
        {{1, 128, 9, 1}, {1, 128, 1, 30}},